prinz_h2o/src/main/scala/pl/touk/nussknacker/prinz/h2o/repository/H2OModelRepository.scala

```diff
@@ -2,20 +2,31 @@ package pl.touk.nussknacker.prinz.h2o.repository
 
 import pl.touk.nussknacker.prinz.h2o.H2OConfig
 import pl.touk.nussknacker.prinz.h2o.model.H2OModel
-import pl.touk.nussknacker.prinz.model.{Model, ModelName}
 import pl.touk.nussknacker.prinz.model.repository.ModelRepository
-import pl.touk.nussknacker.prinz.util.repository.client.{RepositoryClient}
+import pl.touk.nussknacker.prinz.model.{Model, ModelName}
+import pl.touk.nussknacker.prinz.util.repository.client.RepositoryClient
 import pl.touk.nussknacker.prinz.util.repository.payload.ModelPayload
 
 class H2OModelRepository(implicit val config: H2OConfig)
-  extends ModelRepository with RepositoryClient {
+  extends ModelRepository
+    with RepositoryClient {
 
   override def listModels: RepositoryResponse[List[H2OModel]] =
-    client.listModelFiles.right.map(it => it.map(p => new H2OModel(mapPayload(p))).toList)
+    client.listModelFiles.right
+      .map(payloads => payloads
+        .map(mapPayload)
+        .map(new H2OModel(_))
+        .toList
+      )
 
-  override def getModel(name: ModelName): RepositoryResponse[Model] = client.listModelFiles
-    .right.map(it => it.map(mapPayload).filter(p => p.name == name.toString))
-    .map(it => new H2OModel(it.head))
+  override def getModel(name: ModelName): RepositoryResponse[Model] = {
+    client.listModelFiles
+      .right.map(payloads => payloads
+      .map(mapPayload)
+      .filter(p => p.name == name.internal)
+    )
```

Comment:
I think this bracket is incorrectly formatted ;)

Reply:
fixed now i think